.github/workflows/functional-test-cloud.yaml

```diff
@@ -593,7 +631,26 @@ jobs:
       - name: Publish Terraform test recipes
         run: |
           make publish-test-terraform-recipes
-
+      - name: Generate test bicepconfig.json
+        run: |
+          if [[ "${{ env.REL_VERSION }}" == "edge" ]]; then
+            RADIUS_VERSION="latest"
+          else
+            RADIUS_VERSION="${{ env.REL_VERSION }}"
+          fi
+          cat <<EOF > ./test/bicepconfig.json
```

Comment:
We need a temporary bicepconfig.json that will read from a PR-specific tag. So we create one here that goes into the test folder so all our functional tests will use the ACR endpoints in the generated bicepconfig

Reply:
There's already a `bicepconfig.json` in the repo so that'll be used if you're running the tests locally